worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -424,30 +458,10 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
         var isHdfsExpired = false
         if (removedFileInfos != null) {
           removedFileInfos.asScala.foreach {
-            case (_, fileInfo) => {
-              if (fileInfo.isHdfs) {
+            case (_, fileInfo) =>
+              if (cleanFileInternal(shuffleKey, fileInfo)) {
                 isHdfsExpired = true
```

Comment:
"isHdfsExpired = true" should be removed. when fileInfo.isHdfs , isHdfsExpired can be setted to true.

" if (cleanFileInternal(shuffleKey, fileInfo)) {
                isHdfsExpired = true" => cleanFileInternal(shuffleKey, fileInfo)